app/src/pages/inside/profilePage/modals/deleteAccountModal/deleteAccountModal.jsx

```diff
@@ -0,0 +1,111 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { defineMessages, useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import { redirect } from 'redux-first-router';
+import classNames from 'classnames/bind';
+import { ModalLayout, withModal } from 'components/main/modal';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { logoutAction } from 'controllers/auth';
+import { Input } from 'components/inputs/input';
+import { ACCOUNT_REMOVED_PAGE } from 'controllers/pages';
+import styles from './deleteAccountModal.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  header: {
+    id: 'DeleteAccountModal.header',
+    defaultMessage: 'Delete account',
+  },
+  firstDescription: {
+    id: 'DeleteAccountModal.firstDescription',
+    defaultMessage:
+      'The process is irrevocable. By clicking on the <b>"Delete"</b> button you agree to remove all <b>your personal information including your account name, email and photo</b> from our database.',
+  },
+  secondDescription: {
+    id: 'DeleteAccountModal.secondDescription',
+    defaultMessage:
+      'All the data that you have created or reported to ReportPortal will remain in the app, but will no longer be available to you. This includes any <b>launches, filters, widgets, dashboards, etc</b>.',
+  },
+  question: {
+    id: 'DeleteAccountModal.question',
+    defaultMessage: 'Are you sure you want to delete your account?',
+  },
+  label: {
+    id: 'DeleteAccountModal.label',
+    defaultMessage: 'Type "<b>DELETE</b>" to confirm',
+  },
+});
+
+const DELETE = 'DELETE';
+
+// eslint-disable-next-line no-unused-vars
+const DeleteAccount = ({ data }) => {
+  const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
+  const [value, setValue] = useState('');
+
+  const onChange = (e) => {
+    const currentValue = e.target.value;
+    setValue(currentValue);
+  };
+
+  const onDelete = (closeModal) => {
+    closeModal();
+    // todo send 'data' to GA4
+    // todo dispatch delete account action with delete request
+    dispatch(logoutAction());
+    dispatch(redirect({ type: ACCOUNT_REMOVED_PAGE }));
+  };
+
+  const deleteButton = {
+    text: formatMessage(COMMON_LOCALE_KEYS.DELETE),
+    onClick: onDelete,
+    danger: true,
+    disabled: value !== DELETE,
+  };
+  const cancelButton = {
+    text: formatMessage(COMMON_LOCALE_KEYS.CANCEL),
+  };
+
+  return (
+    <ModalLayout
+      title={formatMessage(messages.header)}
+      okButton={deleteButton}
+      cancelButton={cancelButton}
+    >
+      <div className={cx('description')}>{Parser(formatMessage(messages.firstDescription))}</div>
+      <div className={cx('description')}>{Parser(formatMessage(messages.secondDescription))}</div>
```

Comment:
I'm not sure, but what if will decide to change the order of that descriptions?
You'll force to update all of the message names, including localization files.
So I suggest to name it more meaningful according to its content.

Reply:
Fixed.